New package: Crux v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Crux
- Repository: https://github.com/sisl/Crux.jl
- Created by: @ancorso
- Version: v0.1.0
- Commit: 45a17ee16d947a89d15fa0cfac67bd862d6ae788
- Reviewed by: @ancorso
- Reference: https://github.com/sisl/Crux.jl/commit/45a17ee16d947a89d15fa0cfac67bd862d6ae788#commitcomment-137497169
- Description: Julia library for deep reinforcement learning
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd66813b7a4198e2266eedc7b01449237329f6d177f39813a5f55e93257c280b6485216b3d97f1601109a8bed63be8e26b058451503cde6e0ccac6a0787bad6d839634d2407575276ef40333e5ec1ce052ba29300371005107c26b500941ccfcdb78fc9b47f2434360ff0be0f862c8e412b86ab59f53e211f05e0025bffd8e6c67cb0c5b3cf4ce19070074ce138854bd2fd26a84ad8c781891bd97da9752bd05139 -->

goerz commented:
I'm going to assume the name similarity to `Flux` is somewhat intended. So, the name seems fine to me, giving the context of existing packages in the ML space.

ancorso commented:
Yep, you are correct about the name.

JuliaRegistrator committed:
New package: Crux v0.1.0

UUID: e51cc422-768a-4345-bb8e-2246287ae729
Repo: https://github.com/sisl/Crux.jl.git
Tree: c582773fccb99fd7b804d5bc99ebfe35b80162d6

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

ancorso commented:
@goerz It looks like everything except the name checks are now passing. I'd like to keep the name as is based on the reasoning you mentioned in your comment. Is it possible to merge?

goerz commented:
I would merge it, but I don't have maintainer status on the Registry. You'll have to ask on Slack in the `#pgk-registration` channel to get the attention of a maintainer.
